packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -254,6 +257,7 @@ export interface DropdownState {
   value: ShorthandCollection<DropdownItemProps>;
   itemIsFromKeyboard: boolean;
   isFromKeyboard: boolean;
+  inverted: boolean;
```

Comment:
`inverted` should not be part of state, only part of the props.

Reply:
```suggestion
```

Reply:
fixed